tests/api/test_rundb.py

```diff
@@ -115,23 +115,47 @@ def test_update_run(db: RunDBInterface):
 
 
 def test_artifacts(db: RunDBInterface):
-    k1, u1, art1 = "k1", "u1", {"a": 1}
-    db.store_artifact(k1, art1, u1)
-    art = db.read_artifact(k1, u1)
-    assert art1["a"] == art["a"], "get artifact"
+    k1, k2, k3 = "k1", "k2", "k3"
+    t1, t2, t3 = "t1", "t2", "t3"
+    art1 = {
+        "metadata": {
+            "key": k1,
+            "tree": t1,
+            "description": 1,
+        }
+    }
+    db.store_artifact(k1, art1, tree=t1)
+    art = db.read_artifact(k1, tree=t1)
+    assert (
+        art1["metadata"]["description"] == art["metadata"]["description"]
+    ), "get artifact"
     art = db.read_artifact(k1)
```

Comment:
shouldn't read_artifact return an artifact object?

Reply:
you mean instead of a `dict`?